yarn.lock

```diff
@@ -957,13 +957,20 @@
     "@babel/helper-validator-option" "^7.16.7"
     "@babel/plugin-transform-typescript" "^7.16.7"
 
-"@babel/runtime@^7.0.0", "@babel/runtime@^7.10.1", "@babel/runtime@^7.11.1", "@babel/runtime@^7.11.2", "@babel/runtime@^7.12.0", "@babel/runtime@^7.12.5", "@babel/runtime@^7.14.5", "@babel/runtime@^7.4.4", "@babel/runtime@^7.5.5", "@babel/runtime@^7.7.2", "@babel/runtime@^7.8.4", "@babel/runtime@^7.8.7":
+"@babel/runtime@^7.0.0", "@babel/runtime@^7.10.1", "@babel/runtime@^7.11.1", "@babel/runtime@^7.11.2", "@babel/runtime@^7.12.5", "@babel/runtime@^7.14.5", "@babel/runtime@^7.4.4", "@babel/runtime@^7.5.5", "@babel/runtime@^7.7.2", "@babel/runtime@^7.8.4", "@babel/runtime@^7.8.7":
   version "7.16.7"
   resolved "https://registry.yarnpkg.com/@babel/runtime/-/runtime-7.16.7.tgz#03ff99f64106588c9c403c6ecb8c3bafbbdff1fa"
   integrity sha512-9E9FJowqAsytyOY6LG+1KuueckRL+aQW+mKvXRXnuFGyRAyepJPmEo9vgMfXUA6O9u3IeEdv9MAkppFcaQwogQ==
   dependencies:
     regenerator-runtime "^0.13.4"
 
+"@babel/runtime@^7.14.6", "@babel/runtime@^7.17.2":
```

Comment:
Starting to get quite a few duplicates here. Can we run `yarn-deduplicate` or similar? Here is output of `yarn why`:

```
=> Found "@babel/runtime@7.17.7"
info Has been hoisted to "@babel/runtime"
info Reasons this module exists
   - "workspace-aggregator-6660f4f1-831c-4eef-adee-5a4964da2e5a" depends on it
   - Hoisted from "_project_#terriajs#rc-slider#@babel#runtime"
   - Hoisted from "_project_#terriajs#react-i18next#@babel#runtime"
   - Hoisted from "_project_#terriajs#react-select#@babel#runtime"
   - Hoisted from "_project_#terriajs#react-transition-group#@babel#runtime"
   - Hoisted from "_project_#terriajs#rc-slider#rc-tooltip#@babel#runtime"
   - Hoisted from "_project_#terriajs#rc-slider#rc-util#@babel#runtime"
   - Hoisted from "_project_#terriajs#react-select#@emotion#core#@babel#runtime"
   - Hoisted from "_project_#@babel#preset-env#@babel#plugin-transform-regenerator#regenerator-transform#@babel#runtime"
   - Hoisted from "_project_#terriajs#react-transition-group#dom-helpers#@babel#runtime"
   - Hoisted from "_project_#terriajs#rc-slider#rc-tooltip#rc-trigger#@babel#runtime"
   - Hoisted from "_project_#terriajs#rc-slider#rc-tooltip#rc-trigger#rc-align#@babel#runtime"
   - Hoisted from "_project_#terriajs#rc-slider#rc-tooltip#rc-trigger#rc-motion#@babel#runtime"
   - Hoisted from "_project_#terriajs#react-select#@emotion#css#babel-plugin-emotion#babel-plugin-macros#@babel#runtime"
info Disk size without dependencies: "844KB"
info Disk size with unique dependencies: "888KB"
info Disk size with transitive dependencies: "888KB"
info Number of shared dependencies: 1
=> Found "fetch-mock#@babel/runtime@7.17.9"
info This module exists because "_project_#terriajs#fetch-mock" depends on it.
info Disk size without dependencies: "844KB"
info Disk size with unique dependencies: "888KB"
info Disk size with transitive dependencies: "888KB"
info Number of shared dependencies: 1
=> Found "i18next-browser-languagedetector#@babel/runtime@7.18.9"
info This module exists because "_project_#terriajs#i18next-browser-languagedetector" depends on it.
info Disk size without dependencies: "880KB"
info Disk size with unique dependencies: "924KB"
info Disk size with transitive dependencies: "924KB"
info Number of shared dependencies: 1
=> Found "i18next#@babel/runtime@7.18.9"
info This module exists because "_project_#terriajs#i18next" depends on it.
info Disk size without dependencies: "880KB"
info Disk size with unique dependencies: "924KB"
info Disk size with transitive dependencies: "924KB"
info Number of shared dependencies: 1
✨  Done in 0.84s.
```

Reply:
Would like to hear your opinion, but I can put this is a separate ticket for something that could be done repo-wide, wont hold up approval.

Reply:
We probably should have it as a separate ticket as it should be done both TerriaJS and TerriaMap, yarn is on v3 now :D

Reply:
OK Ill make one :-)